pkg/okctl/okctl.go

```diff
@@ -171,31 +171,18 @@ func (o *Okctl) StateHandlers(nodes *clientCore.StateNodes) *clientCore.StateHan
 	}
 }
 
-// InitializeToolChain with core services
-func (o *Okctl) InitializeToolChain() (*clientDirectAPI.ToolChain, error) {
+// InitializeDirectClients with core services
+func (o *Okctl) InitializeDirectClients() (*clientDirectAPI.ToolChain, error) {
 	return &clientDirectAPI.ToolChain{
```

Comment:
Maybe rename this since the toolchain is located on the okctl struct now. You can probably remove it completly

Reply:
It will be removed in a later PR :)